Monika After Story/game/pong.rpy

```diff
@@ -429,40 +604,503 @@ init -1 python in mas_pong:
+    elif instant_loss_streak_counter_before >= 3 and persistent._mas_pm_ever_let_monika_win_on_purpose:
+        m 2tsu "So you're playing seriously now?"
+        m 2tfu "Let's find out how good you really are, [player]!"
+
+    #Player wins after losing at least three times in a row
+    elif loss_streak_counter_before >= 3:
+        m 4eub "Congrats, [player]!"
+        m 2hub "I knew you would win a game after enough practice!"
+        m 4eua "Remember that skill comes mostly through repetitive training."
+        m 4hub "If you train long enough I'm sure you can reach everything you aim for!"
+
+    #Monika loses after saying she would win this time
+    elif powerup_value_this_game == PONG_DIFFICULTY_POWERUP:
+        m 2wuo "Wow..."
+        m 3wuo "I was really trying that time!"
+        m 1hub "Way to go, [player]!"
+
+    #Monika loses after going easy on the player
+    elif powerup_value_this_game == PONG_DIFFICULTY_POWERDOWN:
+        m 1hua "Ehehe!"
+        m 2hub "Good job, [player]!"
+
+    #Monika loses after going even easier on the player
+    elif powerup_value_this_game == PONG_PONG_DIFFICULTY_POWERDOWNBIG:
+        m 1hua "I'm glad you won this time, [player]."
+
+    #Monika loses by a trickshot
+    elif pong_angle_last_shot >= 0.9 or pong_angle_last_shot <= -0.9:
+        if pong_monika_last_response_id == PONG_MONIKA_RESPONSE_LOSE_TRICKSHOT:
+            m 2wuo "[player]!"
+            m 2hksdlb "There's no way I could've hit that!"
+        else:
+            m 2wuo "Wow, there's no way I could've hit that!"
+
+        $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_LOSE_TRICKSHOT
+
+    #Monika loses three times in a row
+    elif win_streak_counter == 3:
+        m 2wuo "Wow, [player]..."
+        m 2wud "You've won three times in a row already..."
+
+        #Easy
+        if pong_difficulty_before <= 5:
+            m 2tsu "Maybe I'm going a little bit too easy on you~"
+
+        #Medium
+        elif pong_difficulty_before <= 10:
+            m 4hua "You're pretty good!"
+
+        #Hard
+        elif pong_difficulty_before <= 15:
+            m 3hub "Well played!"
+
+        #Expert
+        elif pong_difficulty_before <= 20:
+            m 4wuo "That was amazing!"
+
+        #Extreme
+        else:
+            m 2wuo "Wow!"
+            m 2wuw "You beat me three times and I'm giving it all I have!"
+            m 2hub "Great job, [player]!"
+            m 1kua "Ehehe!"
+
+    #Monika loses five times in a row
+    elif win_streak_counter == 5:
+        m 2wud "[player]..."
+        m 2tsu "Have you been practicing?"
+        m 3hksdlb "I don't know what happened but I don't stand a chance against you!"
+        m 1eka "Could you go a little bit easier on me please?"
+        m 3hub "I would really appreciate it~"
+        $ monika_asks_to_go_easy = True
+
+    #Monika loses a long game
+    elif ball_paddle_bounces > 10 and ball_paddle_bounces > pong_difficulty_before * 1/2:
```

Comment:
so I am assuming this should be 1/2 (0.5) times `pong_difficulty_before`, correct?
If so, just use 0.5 as integer precision here probably isn't needed.